data/json/items/comestibles/dairy.json

```diff
@@ -63,41 +65,43 @@
     "id": "cheese_hard",
     "name": "hard cheese",
     "name_plural": "hard cheese",
-    "weight": 120,
+    "weight": 34,
     "color": "yellow",
     "spoils_in": "180 days",
     "stim": 1,
     "container": "wrapper",
     "comestible_type": "FOOD",
     "symbol": "%",
     "quench": -5,
-    "calories": 260,
+    "calories": 135,
     "description": "Hard, dry cheese made to last, unlike modern processed cheese.  Will make you thirsty though.",
     "price": 900,
     "material": [ "milk" ],
     "volume": 1,
-    "charges": 3,
+    "charges": 7,
```

Comment:
Because you changed the charges of this item, you need to divide the weight and price by the number of charges.

In alcohol.json, you change charges on lines (other than this one):
102